### PR TITLE
Replace icons

### PR DESCRIPTION
**Pull request by DavidBachmann**

### CODE REVIEW

File: packages/tux/src/components/fields/DatePicker.tsx [hunk 1]
```diff
@@ -95,12 +96,14 @@ class DatePicker extends Component<any, State> {
           onFocus={this.handleInputFocus}
           onBlur={this.handleInputBlur}>
           <input
+            id="dateInput"
```
Comment (thorsteinsson) [inline packages/tux/src/components/fields/DatePicker.tsx]: We need to generate the ID to support having many components in the same page. This is a global variable.

File: packages/tux/src/components/fields/DatePicker.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import moment from 'moment'
 import DayPicker, { DateUtils } from 'react-day-picker'
 import { Theme, input, button } from '../../theme'
 import { tuxDatePickerStyles } from './DatePicker.styles'
+import { FaCalendar  } from 'react-icons/lib/fa'
```
Comment (thorsteinsson) [inline packages/tux/src/components/fields/DatePicker.tsx]: Minor:  remove one space after `FaCalendar`

File: packages/tux/src/components/fields/BrowseField.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react'
+import { FaImage  } from 'react-icons/lib/fa'
```
Comment (thorsteinsson) [inline packages/tux/src/components/fields/BrowseField.tsx]: Minor: remove one space after `FaImage`

File: packages/tux/src/components/fields/DatePicker.tsx (hunk 1 above)
Comment (eirikurn) [reply packages/tux/src/components/fields/DatePicker.tsx]: Editor components get an id which they should use, so they work with the label, which is rendered uniformly.

**Pull request merged**